specification/cognitiveservices/OpenAI.Inference/tspconfig.yaml

```diff
@@ -21,9 +21,9 @@ options:
     enable-sync-stack: true
     generate-tests: false
   "@azure-tools/typespec-ts":
-    emitter-output-dir: "{project-root}/generated"
     generateMetadata: false
     generateTest: false
     isModularLibrary: true
+    relative-generated-folder: "sources/generated"
```

Comment:
@deyaaeldeen Not sure you could try the command [link](https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/TypeSpec-Project-Scripts.md#typespec-project-processps1) in local to verify if the option works or not.

```
./eng/common/scripts/TypeSpec-Project-Process.ps1 /home/rc/repos/tmp/spec-repo/specification/contosowidgetmanager/Contoso.WidgetManager 677e272f33a3eaa724abd769af79383a5ac2bba5 https://github.com/Azure/azure-rest-api-specs

```

Reply:
It is being added in https://github.com/Azure/azure-sdk-for-js/pull/26138 and I verified it works.